tests/test_config.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+
+from wemake_python_styleguide.config import ConfigFileParser
+
+
+def test_config_parsing(absolute_path):
```

Comment:
Nice! But we also need to test `cli` values.
That can be done in the similar way as we test our code with `Popen`.